go/riscv64.patch

```diff
@@ -1,15 +1,24 @@
 diff --git PKGBUILD PKGBUILD
-index cece4cc..d368d3c 100644
+index 766b9c6..73fd1d0 100644
 --- PKGBUILD
 +++ PKGBUILD
-@@ -29,9 +29,14 @@
- sha256sums=('bfa36bf75e9a1e9cbbdb9abcf9d1707e479bd3a07880a8ae3564caee5711cb99'
+@@ -24,14 +24,22 @@ makedepends=(git go)
+ replaces=(go-pie)
+ provides=(go-pie)
+ options=(!strip staticlibs)
+-source=(https://go.dev/dl/go${pkgver}.src.tar.gz{,.asc})
++source=(https://go.dev/dl/go${pkgver}.src.tar.gz{,.asc}
++        riscv64_cacheline.patch)
```

Comment:
by "add some prefix" I mean distinguishing it from sources of other packages, sorry about the confusion. a `go-riscv64-cachelinepad.patch` may work.